src/modules/expression-evaluator/src/resolvers/buildObject.test.ts

```diff
@@ -0,0 +1,123 @@
+import evaluateExpression from '../evaluateExpression'
+
+test('Testing basic buildObject', () => {
+  const testIn = {
+    operator: 'buildObject',
+    properties: [
+      {
+        key: 'key',
+        value: 'value',
+      },
+    ],
+  }
+  const testOut = {
+    key: 'value',
+  }
+  return evaluateExpression(testIn).then((result: any) => {
+    expect(result).toEqual(testOut)
+  })
+})
+
+test('Testing buildObject for handling erronouse input', () => {
```

Comment:
`erronouse` ? Also I don't really like the tests using:
`{        key: 'key',        value: 'value',      }` because it's so abstract that makes a not in the head. Could you change the actual key and value to something else please?

Reply:
745d643c3dee8ea136dd7377ee0c021562db37aa